extensions/wikia/FacebookPreferences/FacebookPreferencesHooks.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Wikia\DependencyInjection\Injector;
+
+class FacebookPreferencesHooks {
+	public static function onGetPreferences( User $user, array &$preferences ): bool {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$user". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)